specification/src/main/asciidoc/platform/Profiles.adoc

```diff
@@ -9,6 +9,8 @@ The Jakarta EE Web Profile Specification,
 published in conjunction with the present specification, defines the
 first Jakarta EE profile, the Web Profile.
 
+The Jakarta EE Core Profile Specification is a minimal profile that is published separately from the Web Profile and this specification.
```

Comment:
Just to be clear, should you say "..separately from the Web Profile and this **Platform** specification."

Reply:
Fixed